phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexScrutinyMapper.java

```diff
@@ -54,6 +58,8 @@
 
 import com.google.common.base.Joiner;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
JUnit classes generally shouldn't be imported into non-test code, plus this looks unused.